web/src/helpers/translation.js

```diff
@@ -27,29 +27,28 @@ export function translate() {
   return translateWithLocale.apply(null, args);
 }
 
-export function getFullZoneName(zoneCode) {
-  const zoneName = translate(`zoneShortName.${zoneCode}.zoneName`);
-  if (!zoneName) {
-    return zoneCode;
-  }
-  const countryName = translate(`zoneShortName.${zoneCode}.countryName`);
-  if (!countryName) {
-    return zoneName;
-  }
-  return `${zoneName} (${countryName})`;
-}
 
-export function getShortZoneName(zoneCode, limit = 40) {
-  const zoneName = translate(`zoneShortName.${zoneCode}.zoneName`);
+export const getZoneName = (zoneCode) => translate(`zoneShortName.${zoneCode}.zoneName`);
+export const getCountryName = (zoneCode) => translate(`zoneShortName.${zoneCode}.countryName`);
+
+/**
+ * Gets the full name of a zone with the country name in parentheses.
+ * Zone name can be ellipsified with the limit parameter.
+ * @param {string} zoneCode 
+ * @param {number} limit 
```

Comment:
@param {number} [limit=0] if we want to show that there is a default value and that it's optional. However, the description also indicates that it's optional